kolibri/auth/api.py

```diff
@@ -366,23 +366,17 @@ def get_session(self, request):
                    'full_name': user.full_name,
                    'user_id': user.id,
                    'can_manage_content': user.can_manage_content}
-        roles = Role.objects.filter(user_id=user.id)
+
+        roles = list(Role.objects.filter(user_id=user.id).values_list('kind', flat=True).distinct())
```

Comment:
if you are in the `if isinstance(user, AnonymousUser) === False` bloc, can you safely assume that `user.roles` is defined?

Reply:
oh yeah, and i think you're right that with Coach roles scoped to the Classroom + Facility, a user could have more than one role, contradicting what I was saying below.

Reply:
the query will return an empty array if there are no roles defined for the user